docs/adrs/minimal-lifecycle.md

```diff
@@ -135,29 +197,40 @@ Some considered alternatives
 
 #### Minimality
 
-A mutual close is not essential for a complete life-cycle. This will be left for
+A mutual close is not essential for a complete lifecycle. This will be left for
 a future ADR. The add step will also be left for a future ADR.
 
 It's been mooted that close includes the approved output address.
 
 #### Unilateral-ness
```

Comment:
Side note: it is pretty interesting flow when you read the design and then you read critical piece of the motivation behind it ;-)